### DISCUSSION TITLE
请问libPag能否在英特尔的GPU上跑

### DISCUSSION THREAD

**Discussion by 327635328** [Tencent/libpag#2226] [General] [answered]
不同显卡上的GPU运行Pag在效果上是否有区别
例如英伟达的A10, T4 , 或者其他英特尔的显卡

**Comment by kevingpqi123** [comment, marked as answer]
对于服务端，默认采用的是 CPU 的渲染模式，如果需要使用 GPU 渲染，需要自己去适配，主要涉及 OpenGL ES GPU 适配和 h264 硬解

**Comment by 327635328** [comment]
已收到
